File: C#/Integrante.cs
using System;

public class Integrante {
	private String nombre;
	private String rol;

	public String GetNombre() {
		return this.nombre;
	}
	public void SetNombre(ref String nombre) {
		this.nombre = nombre;
	}
	public String GetRol() {
		return this.rol;
	}
	public void SetRol(ref String rol) {
		this.rol = rol;
	}
	public Integrante(String nombre, String rol) {
		this.nombre = nombre;
        this.rol = rol;
	}
    public override String ToString() {
		return base.ToString() + "\n" + "Nombre: "+ nombre.ToString() + "\n"+ "Rol: "+ rol.ToString() 
		+ "\n";
	}

}

File: Java/Integrante.java
import java.util.Vector;

public class Integrante {
	private String nombre;
	private String rol;
	public Equipo has;
	public Vector<Tareas> tareas = new Vector<Tareas>();

	public String getNombre() {
		return this.nombre;
	}

	public void setNombre(String aNombre) {
		this.nombre = aNombre;
	}

	public String getRol() {
		return this.rol;
	}

	public void setRol(String aRol) {
		this.rol = aRol;
	}

	public void toString() {
		throw new UnsupportedOperationException();
	}

	public Integrante(String aNombre, String aRol) {
		throw new UnsupportedOperationException();
	}
}